### PR TITLE
Move topic ES6 to EcmaScript

### PR DESCRIPTION
**Pull request by Nixinova**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

----

### Change

Move topic ES6 to EcmaScript.
The description for ES6 was ECMAScript and not ES6 so it is better to have ECMAScript be the primary topic.
ES6 kept as alias.

### CODE REVIEW

File: topics/ecmascript/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+topic: ecmascript
+display_name: ECMAScript
```
Comment (itsbagpack) [inline topics/ecmascript/index.md:3]: Could we please keep these key-values alphabetized? This will be in line with most other topics in the repo.

**Review by itsbagpack — COMMENTED** [review]
Seems like a valuable change. I'll approve this PR once the `items` are alphabetized. Thank you!

File: topics/ecmascript/index.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 ---
-topic: ecmascript
-display_name: ECMAScript
 aliases: es6, es6-javascript, es6-tutorial
 created_by: Brendan Eich, Ecma International
+display_name: ECMAScript
 github_url: https://github.com/tc39
 logo: ecmascript.png
 short_description: ECMAScript is the standards organization behind JavaScript.
+topic: ecmascript
```
Comment (Vannsdok) [inline topics/ecmascript/index.md:8]:
```suggestion
topic: ecmascript
```https://github.com/github/explore/pull/2455/files/e4270e345b968ae626310bc86e339a2ae80c6ae4..557eeba753a7732ef5ac26057a507c8f97717272#diff-0a55323c4c4a3701c4ac4465ca4f58f6f0802a1f431185c89f6e69d13b6d9ce3

File: topics/ecmascript/index.md (hunk 1 above)
Comment (Vannsdok) [reply topics/ecmascript/index.md]: ECMAScript

File: topics/ecmascript/index.md (hunk 2 above)
Comment (Vannsdok) [reply topics/ecmascript/index.md:8]: *  @github/communities-oss-reviewers

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

**Pull request merged**